script/jenkins/test-install-packages.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -xeuo pipefail
 
-mage -v -debug testPackagesInstall
+make test-packages-install
```

Comment:
```suggestion
./build/linux/mage test-packages-install
```

I'm not too keen on adding something like this to the Makefile, unless it's something we run regularly in day-to-day development. I didn't even know it existed until now, but I don't know - do you run this regularly yourself?

Reply:
I don't.